### PR TITLE
convert gdigg to digg and withdraw exactly what is needed from tcl

### PR DESCRIPTION
**Pull request by jorijnsmit**
this is to have enough $digg in trops for the whole of q2, both for slp wbtcdigg and bdigg emissions. issue #330

### CODE REVIEW

**Review by jalbrekt85 — DISMISSED** [review]
`12.5 * 13` gdigg -> `21.164` digg to be withdrawn

nice use of `get_lp_to_withdraw_given_token` for exact amount

```
snapshot result for 0x042B32Ac6b453485e357938bdC38e0340d4b9276:
              balance_before          balance_after          balance_delta
symbol                                                                    
WBTC    2.819281630000000000  16.916807020000000000  14.097525390000000000
DIGG    4.269945880000000000  25.434904612000000000  21.164958732000000000 
```
`21.164` digg to` trops `reflected in snapshot, lgtm

**Review by jalbrekt85 — DISMISSED** [review]

File: scripts/issue/330/withdraw_digg_from_tcl_q2.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+    trops = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    wbtc = interface.ERC20(registry.eth.treasury_tokens.WBTC)
+    digg = interface.IUFragments(registry.eth.treasury_tokens.DIGG)
+    slp_wbtcdigg = safe.contract(registry.eth.treasury_tokens.slpWbtcDigg)
+
+    safe.init_badger()
+    # https://forum.badger.finance/t/bip-88-emissions-q2-2022/5533/9
+    # 13 weeks in q2
+    total_mantissa = safe.badger.from_gdigg_to_digg(12.5 * 13) * Decimal(1e9)
```
Comment (petrovska-petro) [inline scripts/issue/330/withdraw_digg_from_tcl_q2.py:24]: i believe here should be `10 * 13` instead as for `bDIGG` it is 100% auto compounded

Comment (petrovska-petro) [reply scripts/issue/330/withdraw_digg_from_tcl_q2.py:24]: not so much related with this PR, but what will be the new home for the wd WBTC?

still wbtc/badger curve v2 represents a good deal i think, no?

Comment (jorijnsmit) [reply scripts/issue/330/withdraw_digg_from_tcl_q2.py:24]: that autocompounded digg has to come from somewhere though ;) it doesnt get minted anymore, but is added to the vault by the rewards (geyser) contract when harvesting; eg see https://etherscan.io/address/0x5b60952481eb42b66bdffc3e049025ac5b91c127#tokentxns

**Review by petrovska-petro — APPROVED** [review]

**Review by jalbrekt85 — APPROVED** [review]

**Pull request merged**